python/cuml/cuml/fil/fil.pyx

```diff
@@ -1099,7 +1085,14 @@ class ForestInference(UniversalBase, CMajorInputTagMixin):
         message='ForestInference.predict_proba',
         domain='cuml_python'
     )
-    def predict_proba(self, X, *, preds=None, chunk_size=None) -> CumlArray:
+    def predict_proba(
+        self,
+        X,
+        *,
+        preds=None,
+        chunk_size=None,
+        safe_dtype_conversion=None,
```

Comment:
The `safe_dtype_conversion` parameter was removed in 25.04 with the promotion of experimental FIL and the demotion of legacy FIL without deprecation. We should not re-introduce it here.

I had also removed it in https://github.com/rapidsai/cuml/pull/6728 .